keyboards/gopolar/gg86/gg86.c

```diff
@@ -0,0 +1,79 @@
+/* Copyright 2021 Gopolar
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "gg86.h"
+
+// OLED animation
+#include "lib/logo.c"
```

Comment:
`c` files should NOT be included, but should be added by the `rules.mk` file using `SRC += lib/logo.c`.

```suggestion
```

Reply:
Weird, it gave me compile error. After moving on rules.mk files

![image](https://user-images.githubusercontent.com/24840279/156012186-4f161408-085b-4e29-92e3-66fe380a9d2b.png)

Reply:
The logo.c probably is missing important includes then.

Reply:
Generally it's not a good idea to include .c files in other .c files, as it can potentially lead to multiple definition errors. Use header files to declare symbols you need to use and include that instead, so that the implementation is only compiled once.